packages/opam-check-npm-deps/opam-check-npm-deps.1.0.0/opam

```diff
@@ -0,0 +1,53 @@
+opam-version: "2.0"
+synopsis:
+  "An opam plugin to check for npm depexts inside the node_modules folder"
+description:
+  "Provides the `opam check-npm-deps` command, which given an opam switch, gathers all the depexts belonging to the npm platform and their version constraints, and checks the `node_modules` folder to see if the constraints are satisfied."
+maintainer: ["Javier Chávarri"]
+authors: ["Javier Chávarri"]
+license: "MIT"
+homepage: "https://github.com/jchavarri/opam-check-npm-deps"
+bug-reports: "https://github.com/jchavarri/opam-check-npm-deps/issues"
+depends: [
+  "ocaml" {>= "4.14.0"}
+  "reason" {>= "3.8.1" & < "4.0.0"}
+  "dune" {>= "3.8"}
+  "opam-client" {>= "2.1.2" & < "2.2.0"}
+  "mccs" {>= "1.1+14"}
+  "angstrom" {>= "0.15.0"}
+  "fmt" {>= "0.9.0"}
+  "bos"
+  "lwt_ppx"
+  "ppx_deriving_yojson"
+  "ppx_expect"
+  "ppx_inline_test"
+  "ppx_let"
+  "ppx_sexp_conv"
+  "ocaml-lsp-server" {dev}
+  "ocamlformat" {dev}
+  "odoc" {with-doc}
+]
```

Comment:
This is necessary to avoid having the client mismatch the version of the library requested as dependency and slowing up both in some cases: https://github.com/ocaml/opam/issues/4554. There might also be similar unpatched issues so i think we should be careful here
```suggestion
]
available: opam-version >= "2.1" & opam-version < "2.2"
```